rest/resource-server/src/main/java/org/eclipse/sw360/rest/resourceserver/license/LicenseController.java

```diff
@@ -309,7 +313,7 @@ public ResponseEntity<?> uploadLicenses(
             description = "Import OSADL information.",
             tags = {"Licenses"}
     )
-    @RequestMapping(value = LICENSES_URL + "/import/OSADL", method = RequestMethod.POST)
+    @RequestMapping(value = LICENSES_URL + "/import/OSADL", method = RequestMethod.GET)
```

Comment:
Why do we need to change from **POST** to **GET**. I think POST is more suitable.